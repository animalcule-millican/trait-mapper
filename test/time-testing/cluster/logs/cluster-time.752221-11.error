Input WT_TR_GHRAS_02.10000000 does not exist
Input /home/glbrc.org/millican/TMPDIR/warm-lynx/seqDB does not exist
Input /home/glbrc.org/millican/TMPDIR/warm-lynx/clusterDB does not exist
