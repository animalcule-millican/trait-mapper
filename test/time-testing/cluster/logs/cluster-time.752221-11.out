createdb WT_TR_GHRAS_02.10000000 /home/glbrc.org/millican/TMPDIR/warm-lynx/seqDB -v 3 

MMseqs Version:       	14.7e284
Database type         	0
Shuffle input database	true
Createdb mode         	0
Write lookup file     	1
Offset of numeric ids 	0
Compressed            	0
Verbosity             	3

cluster /home/glbrc.org/millican/TMPDIR/warm-lynx/seqDB /home/glbrc.org/millican/TMPDIR/warm-lynx/clusterDB tmp --min-seq-id 0.999 --db-load-mode 2 --threads 24 --remove-tmp-files 1 -v 3 

MMseqs Version:                     	14.7e284
Substitution matrix                 	aa:blosum62.out,nucl:nucleotide.out
Seed substitution matrix            	aa:VTML80.out,nucl:nucleotide.out
Sensitivity                         	4
k-mer length                        	0
k-score                             	seq:2147483647,prof:2147483647
Alphabet size                       	aa:21,nucl:5
Max sequence length                 	65535
Max results per query               	20
Split database                      	0
Split mode                          	2
Split memory limit                  	0
Coverage threshold                  	0.8
Coverage mode                       	0
Compositional bias                  	1
Compositional bias                  	1
Diagonal scoring                    	true
Exact k-mer matching                	0
Mask residues                       	1
Mask residues probability           	0.9
Mask lower case residues            	0
Minimum diagonal score              	15
Selected taxa                       	
Include identical seq. id.          	false
Spaced k-mers                       	1
Preload mode                        	2
Pseudo count a                      	substitution:1.100,context:1.400
Pseudo count b                      	substitution:4.100,context:5.800
Spaced k-mer pattern                	
Local temporary path                	
Threads                             	24
Compressed                          	0
Verbosity                           	3
Add backtrace                       	false
Alignment mode                      	3
Alignment mode                      	0
Allow wrapped scoring               	false
E-value threshold                   	0.001
Seq. id. threshold                  	0.999
Min alignment length                	0
Seq. id. mode                       	0
Alternative alignments              	0
Max reject                          	2147483647
Max accept                          	2147483647
Score bias                          	0
Realign hits                        	false
Realign score bias                  	-0.2
Realign max seqs                    	2147483647
Correlation score weight            	0
Gap open cost                       	aa:11,nucl:5
Gap extension cost                  	aa:1,nucl:2
Zdrop                               	40
Rescore mode                        	0
Remove hits by seq. id. and coverage	false
Sort results                        	0
Cluster mode                        	0
Max connected component depth       	1000
Similarity type                     	2
Single step clustering              	false
Cascaded clustering steps           	3
Cluster reassign                    	false
Remove temporary files              	true
Force restart with latest tmp       	false
MPI runner                          	
k-mers per sequence                 	21
Scale k-mers per sequence           	aa:0.000,nucl:0.200
Adjust k-mer length                 	false
Shift hash                          	67
Include only extendable             	false
Skip repeating k-mers               	false

createsubdb /home/glbrc.org/millican/TMPDIR/warm-lynx/clusterDB /home/glbrc.org/millican/TMPDIR/warm-lynx/seqDB /home/glbrc.org/millican/TMPDIR/warm-lynx/repDB --id-mode 1 -v 3 

MMseqs Version: 	14.7e284
Subdb mode      	0
Database ID mode	1
Verbosity       	3

